.travis.yml

```
@@ -20,7 +20,7 @@ addons:
 script:
   - sudo apt-get install docker.io
   - cd openEuler20.03
-  - docker build -t kunpengcompute/openeuler:rpmbuild .
+  - docker build -t kunpengcompute/openeuler:rpmbuild aarch64
```

You can add the arch in L11, and do somethingh like:
```suggestion
      if [ "aarch64" == "$(uname -m)" ]; then
        # build aarch64
      else
        # build x86
      fi;
```

---

updated. Please take a look. Thanks